crd-reference/generate.sh

```diff
@@ -19,14 +21,14 @@ wget https://github.com/spinkube/spin-operator/releases/download/$SPIN_OPERATOR_
 # Generate SpinAppExecutor Reference Docs
 echo "Generating CRD reference docs for SpinAppExecutor"
 crdoc -r ${spin_operator_crds_file} \
-    -o ${root_dir}/content/en/docs/spin-operator/reference/spin-app-executor.md \
+    -o ../content/en/docs/reference/spin-app-executor.md \
```

Comment:
`rootdir` is to protect agains running the script in a directory other than root.  It's not critical but nice to have.

Reply:
I think that `rootdir` is not necessary anymore b/c we're using `script_dir` to cd into the script directory and thus everything is relative to that.